Can't build with 0.3.9 - undefined reference to `gettid'

Hello,
The build fails with the latest release `0.3.9` with the following error:
```
 note: /home/alex/development/sdk-coreagent/target/debug/deps/libspdlog-6f3a8729f365763c.rlib(spdlog-6f3a8729f365763c.spdlog.cca83b2da12e547f-cgu.00.rcgu.o): In function `spdlog::record::get_current_tid::get_current_tid_inner':
          /home/alex/.cargo/registry/src/index.crates.io-6f17d22bba15001f/spdlog-rs-0.3.9/src/record.rs:264: undefined reference to `gettid'
```

OS ubuntu 18.04
Had to force cargo to work with the previous release:
```
spdlog-rs = {version = "=0.3.8", features = ["source-location", "log"]}
```

--- a/spdlog/src/record.rs
+++ b/spdlog/src/record.rs
@@ -261,7 +261,11 @@ fn get_current_tid() -> u64 {
     #[cfg(target_os = "linux")]
     #[must_use]
     fn get_current_tid_inner() -> u64 {
-        let tid = unsafe { libc::gettid() };
+        // https://github.com/SpriteOvO/spdlog-rs/issues/31
+        //
+        // We don't use `gettid` since earlier glibc versions (before v2.30) did not
+        // provide a wrapper for this system call.
+        let tid = unsafe { libc::syscall(libc::SYS_gettid) };
         tid as u64
     }
 

